kalite/updates/api_views.py

```diff
@@ -235,99 +203,14 @@ def delete_language_pack(request):
     return JsonResponse({"success": _("Successfully deleted language pack for %(lang_name)s.") % {"lang_name": get_language_name(lang_code)}})
 
 
-def annotate_topic_tree(node, level=0, statusdict=None, remote_sizes=None, lang_code=None):
```

Comment:
We don't need this any more (this was happening on the fly on every API request) - the data is stamped on content in the database.